nasa/harmony-regression-tests
# HARMONY-1649 - Remove no-op Swath Projector request.

## Description

This PR removes a no-op Swath Projector regression test. HARMONY-1649 added a service that will return the source data download links if there is no specified transformation. The request removed from this notebook did not specify a transformation, and so is now being routed to that service instead of the Swath Projector.

## Jira Issue ID

[HARMONY-1649](https://bugs.earthdata.nasa.gov/browse/HARMONY-1649)

## Local Test Steps

* Pull this branch.
* `cd test`
* Run `make swath-projector-image`
* Add `EDL_USER` and `EDL_PASSWORD` environment variables (set to your own credentials).
* Add `HARMONY_HOST_URL=https://harmony.uat.earthdata.nasa.gov` as an environment variable.
* Run `./run_notebooks.sh swath-projector`.
* The tests should all pass.

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
